<launch>
  
  <node pkg="cotesys_ros_grasping" type="move_arm_to_position_server" name="move_arm_to_position_server" output="screen">
    <remap from="left_move_arm_action" to="move_left_arm"/>  
    <remap from="right_move_arm_action" to="move_right_arm"/>  
    <rosparam command="load" file="$(find cotesys_ros_grasping)/config/pr2_grasps.yaml" />	 
  </node>  
  
  <node pkg="cotesys_ros_grasping" type="move_arm_relative_cartesian_point_server" name="move_arm_relative_cartesian_point_server" output="screen">
    <remap from="left_move_arm_action" to="move_left_arm"/>  
    <remap from="right_move_arm_action" to="move_right_arm"/>  
    <param name="disable_collisions" type="bool" value="true" />    
  </node>  
  
  <node pkg="cotesys_ros_grasping" type="take_static_collision_map_server" name="take_static_collision_map_server_server" output="screen"/>

  <node pkg="cotesys_ros_grasping" type="attach_bounding_box_server" name="attach_bounding_box_server" output="screen">
  	 <param name="link_to_gripper_x_diff" type="double" value=".14" />    
  </node>
</launch>
